kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1893,6 +1893,28 @@ object KyuubiConf {
       .intConf
       .createWithDefault(0)
 
+  val OPERATION_RESULT_SAVE_TO_FILE: ConfigEntry[Boolean] =
+    buildConf("kyuubi.operation.result.save.to.file")
+      .doc("The switch for Spark query result save to hdfs file")
+      .version("1.9.0")
+      .booleanConf
+      .createWithDefault(false)
+
+  val OPERATION_RESULT_SAVE_TO_FILE_PATH: ConfigEntry[String] =
+    buildConf("kyuubi.operation.result.save.to.file.path")
+      .doc("The hdfs path of Spark query result save")
+      .version("1.9.0")
+      .stringConf
+      .createWithDefault("/tmp/kyuubi/tmp_kyuubi_result")
+
+  val OPERATION_RESULT_SAVE_TO_FILE_THRESHOLD: ConfigEntry[Int] =
+    buildConf("kyuubi.operation.result.save.to.file.threshold")
+      .doc("The threshold of Spark result save to hdfs file, default value is 200 MB")
+      .version("1.9.0")
+      .intConf
+      .checkValue(_ > 0, "must be positive value")
+      .createWithDefault(209715200)
```

Comment:
possible to support human-readable strings? e.g. 200MiB, 200MB, 200m, 1G

Reply:
`kyuubiConf` doesn't have `byteConf` like spark, I can submit another pr if need